apps/cookbook/src/app/examples/example-configuration-wrapper/example-configuration-wrapper.component.scss

```diff
@@ -1,7 +1,47 @@
 @use '@kirbydesign/core/src/scss/utils';
 
 :host {
-  display: inline-block;
+  &.toggle {
+    display: flex;
+    flex-direction: column;
+    align-items: end;
+    position: absolute;
+    right: utils.size('s');
+    top: calc(utils.size('s') + var(--ion-safe-area-top));
+
+    &.show-config {
+      kirby-card {
+        display: initial;
+      }
+    }
+
+    kirby-card {
+      display: none;
+    }
+  }
+
+  &.snap-to-viewport {
+    kirby-card {
+      position: fixed;
+      margin-top: utils.$border-radius;
+      opacity: 0.5;
+      transition-property: transform, border-radius;
+      transition-duration: utils.get-transition-duration('short');
+      transition-timing-function: utils.get-transition-easing('enter-exit');
+      transform: rotate(-90deg) translateY(-#{utils.$border-radius + utils.size('xs')});
```

Comment:
Restore previous functionality with no rotation and 40% transform on larger screens